config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get 'merchants/:id/items', to: 'merchant_items#index'
```

Comment:
We may want to change this route based on our lesson from today. I think it may be better to do so now rather than as a refactor later on so we'd have to do less work replacing and tweaking things.

What does everyone think?

Reply:
Agreed

Reply:
Cool. I'll go ahead and merge this for now and let's get together as a group to set up our routes later on.